drivers/SmartThings/philips-hue/src/lunchbox/sse/eventsource.lua

```diff
@@ -275,6 +275,9 @@ local function connecting_action(source)
         })
         if err ~= nil then return nil, err end
 
+        err = select(2, source._sock:settimeout(60))
```

Comment:
Maybe include a comment to explain why it is being set again down here, when it is set above.